src/app/app.component.ts

```diff
@@ -1,9 +1,18 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { AngularFireDatabase } from '@angular/fire/database';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html'
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'ecommerce-angular';
+
+  constructor(private db: AngularFireDatabase) {
+
+  }
+  ngOnInit() {
+    this.db.list('/courses').valueChanges()
```

Comment:
we should never check in console.logs. If you do please leave a //TODO: remove when building courses page